types/ethjs/index.d.ts

```diff
@@ -1,14 +1,18 @@
+declare module '@metamask/ethjs-types' {
```

Comment:
Note that usually type augmentations aren't a good idea because they will only be present locally and will not be included in the type declaration files, so they could cause type errors in another project that uses this package. This seems to be okay for right now because `registry` is private, so `DeployedRegistryContract` gets erased from the built type declaration files, but something to keep in mind in case this does cause problems later.